File: Java/UberPool.java
//Clase hija
class UberPool extends Car {
    String brand;
    String model;
    
    //Construcctor
    public UberPool(
            String licence,
            Account driver,
            String brand,
            String model
        ){
            super(licence, driver);
            this.brand = brand;
            this.model = model;
        }
}

File: Python/UberPool.py
from car import Car

class UberPool(Car):
        brand = str
        model = str
        
        def __int__(
            self,
            license,
            driver,
            brand,
            model
        ):
            super.__init__(license, driver, brand, model)
            self.brand = brand
            self.model = model